assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -492,6 +492,13 @@ $cart-item-label-offset:                $cart-thumbnail-maxWidth + $cart-item-sp
             float: right;
         }
     }
+
+    .checkout-multiple {
```

Comment:
So, I thought that too, but the elements on this page don't seem to follow `camelCase` - the container is `cart-actions` for example, and there are numerous other violations. I'm happy to use the right format, I was just following the surrounds.